test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -64,6 +64,7 @@ def copy_fn(x):
         example_inputs = tuple(copy_fn(x) for x in example_inputs)
 
     torch._dynamo.reset()
+    torch._inductor.metrics.reset()
```

Comment:
Should we just have this run in the setup of all test_torchinductor tests ? https://github.com/pytorch/pytorch/blob/c61025e2b2cfb5f58651f04a64bd2380bebe1c01/test/inductor/test_torchinductor.py#L122

Reply:
Good idea, I think it might also be good to keep it here as well though because I think there are some cases that run common() multiple times